clang/lib/CrossTU/CrossTranslationUnit.cpp

```diff
@@ -569,7 +569,7 @@ CrossTranslationUnitContext::ASTLoader::loadFromDump(StringRef ASTDumpPath) {
   return ASTUnit::LoadFromASTFile(
       ASTDumpPath, CI.getPCHContainerOperations()->getRawReader(),
       ASTUnit::LoadEverything, Diags, CI.getFileSystemOpts(),
-      CI.getHeaderSearchOptsPtr());
+      CI.getHeaderSearchOpts());
```

Comment:
@Xazax-hun  I think this change would mean that LoadFromASTFile would no longer modify the header search options on CI here. I'm not familiar with the CrossTU code; do you know if that is okay?

Reply:
Ping, and CC @gamesh411 too.